.travis.yml

```diff
@@ -16,9 +15,18 @@ install:
         && cd ..
     - |
         cd secp256k1 \
-        && phpize && ./configure --with-secp256k1 && make && sudo make install \
+        && phpize && ./configure && make && sudo make install \
         && cd ..
     - composer update
 
+before_script:
+    - php-config || true
+    - phpenv rehash
+    - phpenv version-name
+
 script:
+    - |
+        cd libsecp256k1 \
+        && make test \
+        && cd ..
```

Comment:
this is tmp, running libsecp256k1 tests to see if it doesn't segfault somewhere in there for w/e reason